src/psyclone/nemo.py

```diff
@@ -472,6 +481,31 @@ def __str__(self):
         result += "EndLoop"
         return result
 
+    def reference_accesses(self, var_accesses):
+        '''Get all variable access information. The loop variable is
+        set as READ and WRITE. Then the loop body's access is added.
+        TODO: The start, stop and step values are only strings, so we
```

Comment:
Please include the issue number for this TODO (and create said issue if needed).